### ISSUE TITLE
Temporary amnesty

### ISSUE THREAD

**Issue by qdacsvx** [iftechfoundation/ifdb-suggestion-tracker#114]
It will take time for people to get used to an actively moderated environment. The morning exchange of obscenities with frenemies will have to go.

People who use one account for publishing reviews and another for publishing games to avoid mutual downvoting will need to tell moderators. You can tell @dfabulich,

Inevitably people will recklessly carry on doing what they always did without reflecting that now somebody is watching who can ban you for bad behaviour and **WILL BAN YOU FOR VIOLATING THE TERMS OF SERVICE**.

Moderators warning people may not be effective if people don't read messages or don't look at email #77. In this situation moderators should be visible by posting Dos-and-Don't site news. People who get the message will communicate with others who aren't looking at the front page news.

Moderators shouldn't assume that the long tail of individuals who login once per week, per 2-weeks, per month, per months, per year; saw a random thread on intfiction.org or even knows there are moderators. There probably should be a login message.

Moderators should ask/warn and try to understand rather than rush to judgement.

If you are removing tags remember that multiple different people tag games. If it seems like someone is fighting by re-adding tags, remember that it could be an innocent third-party tagger who didn't even know the tag was removed. And this could happen multiple times with multiple independent taggers.

**Comment by dfabulich** [comment]
I guess this is a suggestion for moderator policy rather than a proposal for a code change on IFDB?

**Comment by curiousdannii** [comment]
Egregious behaviour shouldn't be excused just because it wasn't quickly enforced previously.

Problematic behaviour should be stopped with a warning with stronger actions if it continues. This too will be how moderation should happen long after the IFTF has taken over. So I don't think an amnesty is needed.

**Comment by qdacsvx** [comment]
@dfabulich Yes this is advice for moderators. Tag removal could be an issue for everyone if everyone can do it.

@curiousdannii Persuade me that users can reliably receive warnings #77 and I will agree.

**Comment by brirush84** [comment]
If most moderation results in reversible actions (hiding a user, for instance), then it should be fine if people don't notice for a while, since once they do log in they can request that the action be reversed.

**Comment by qdacsvx** [comment]
Hiding someone can harm others. What if there is an open question and I answer it. But my answer is hidden so the questioner isn't helped.

**Comment by qdacsvx** [comment]
[Tag removal for admins](url) (proposed) is not reversible.

**Comment by brirush84** [comment]
IFDB doesn't have questions, as it's not a forum (and I don't believe it should become one). If a regular user has the ability to hide things, moderators should be reviewing all such decisions so that no one is inappropriately hidden (I would be happy to share that duty). If only moderators can hide something, then if an answer gets hidden there's a reason for it to be hid.

**Comment by qdacsvx** [comment]
@brirush84 Questions can appear in a review "Did anyone find out how to enter the cave?" or in a comment "I never finished that game I just couldn't find the golden spoon."

In either case, posting an answer to the question would help the person who posted and perhaps others. (Consider putting answers in <spoiler> answer </spoiler> if it solves a puzzle.) Unfortunately if I'm hidden my answers couldn't help anyone.
